### DISCUSSION TITLE
Running single test

### DISCUSSION THREAD

**Discussion by tushargugnani** [pestphp/pest#570] [Q&A]
Is there a way I can run a single test in Pest, same as by using phpunit --filter function? There is an `only` method but that only works when you only have a pest test in your test suite.

**Comment by dansysanalyst** [comment]
You can use `--filter`. Pest will understand it and defer to PHPUnit.

For example:

`./vendor/bin/pest --filter AuthenticationTest`

Results in

```plain
   PASS  Tests\Feature\Jetstream\AuthenticationTest
  ✓ login screen can be rendered
  ✓ users can authenticate using the login screen
  ✓ users cannot authenticate with invalid password

  Tests:  3 passed
```

You can read more [here](https://pestphp.com/docs/cli-options).

**Reply by surgiie** [reply]
This runs all test in a file, how do you run a specific test  under `AuthenticationTest` ?

**Reply by nunomaduro** [reply]
`./vendor/bin/pest path/to/AuthenticationTest --filter="your test description"`
